account_invoice_check_total/tests/test_account_invoice.py

```diff
@@ -1,66 +1,79 @@
 # Copyright 2016 Acsone SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import TransactionCase
 from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
 from ..models.account_invoice import GROUP_AICT
 
 
 class TestAccountInvoice(TransactionCase):
-
     def setUp(self):
         super(TestAccountInvoice, self).setUp()
 
         # ENVIRONEMENTS
 
-        self.account_invoice = self.env['account.invoice']
-        self.account_model = self.env['account.account']
-        self.account_invoice_line = self.env['account.invoice.line']
+        self._account_move = self.env["account.move"]
```

Comment:
Why is this underscore needed?
```suggestion
        self.account_move = self.env["account.move"]
```